atlas/packages/app/src/components/LayoutWithRouting.tsx

```diff
@@ -1,17 +1,25 @@
 import React from 'react'
+import styled from '@emotion/styled'
+import { Router } from '@reach/router'
+
 import { GlobalStyle } from '@/shared/components'
-import { HomeView, VideoView } from '@/views'
+import { BrowseView, HomeView, VideoView } from '@/views'
 import routes from '@/config/routes'
-import { Router } from '@reach/router'
+import sizes from '@/shared/theme/sizes'
```

Comment:
Let's be consistent with the rest of the codebase here and let's do `import {sizes} from '@/shared/theme' ` or import the theme directly

Reply:
Certainly, fixed now